pom.xml

```diff
@@ -1429,7 +1428,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-databind</artifactId>
-                <version>${jackson.version}</version>
+                <version>2.8.11.1</version> <!-- TODO sync. with jackson.version -->
```

Comment:
Why isn't the jackson.version variable in use here?  Could more explanation be added for the choice of this version?

Reply:
Because jackson-databind-2.8.11 still has the bug.  There is no 2.8.11.1 of the other Jackson components.